packages/web3-eth-accounts/src/index.js

```diff
@@ -525,10 +525,35 @@ Wallet.prototype.load = function (password, keyName) {
     return this.decrypt(keystore || [], password);
 };
 
-if (typeof localStorage === 'undefined') {
+if (!storageAvailable('localStorage')) {
     delete Wallet.prototype.save;
     delete Wallet.prototype.load;
 }
 
+function storageAvailable(type) {
+    var storage;
+    try {
+        storage = window[type];
+        var x = '__storage_test__';
+        storage.setItem(x, x);
+        storage.removeItem(x);
+        return true;
+    }
+    catch(e) {
+        return e && (
```

Comment:
The MDN code has `return e instanceof DOMException && (` in this line. Is this equivalent? How?

Reply:
```js
e instanceof DOMException
```
returns a boolean which represents whether `e` is an instance of `DOMException` or not.
The correct way of doing this would be to use a [polyfill for DOMException](https://github.com/inexorabletash/polyfill/blob/master/obsolete/domexception.js) but that would be too much code.

If this code is ever executed in a browser and caught, it would always be a [DOMException](https://developer.mozilla.org/en-US/docs/Web/API/DOMException) because `storage` is part of Web API. Hence, i don't really check whether it's a DOMException or not.

In case of node, window wouldn't be defined and it would be caught and returns false